packages/mariadb-10.11-mroonga/debian/control.in

```diff
@@ -0,0 +1,122 @@
+Source: mariadb-10.11-mroonga
+Section: database
+Priority: optional
+Maintainer: Horimoto Yasuhiro <horimoto@clear-code.com>
+Build-Depends:
+	debhelper (>= 10),
+	autotools-dev,
+	pkg-config,
+	libgroonga-dev (>= @REQUIRED_GROONGA_VERSION@),
+	groonga-normalizer-mysql,
+	libmariadb-client-lgpl-dev-compat,
+	libmariadbd-dev,
+	libssl-dev,
+	wget,
+	lsb-release,
+	bison,
+	chrpath,
+	cmake,
+	dh-apparmor,
+	gdb,
+	libaio-dev [linux-any],
+	libboost-dev,
+	libjemalloc-dev [linux-any],
+	libjudy-dev,
+	libncurses5-dev,
+	libpam0g-dev,
+	libpcre3-dev,
+	libreadline-dev,
+	libsnappy-dev,
+	libxml2-dev,
+	lsb-release,
+	perl,
+	po-debconf,
+	psmisc,
+	unixodbc-dev,
+	zlib1g-dev,
+	autoconf-archive
+Standards-Version: 3.9.1
```

Comment:
We can use 4.6.2 for Debian GNU/Linux bookworm or later: https://packages.debian.org/search?keywords=debian-policy

```suggestion
Standards-Version: 4.6.2
```